scripts/lc-builds/toss4_nvcc_clang.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright (c) 2016-25, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+if [[ $# -lt 3 ]]; then
+  echo
+  echo "You must pass 3 arguments to the script (in this order): "
+  echo "   1) compiler version number for nvcc"
+  echo "   2) CUDA compute architecture (number only, not 'sm_90' for example)"
+  echo "   3) compiler version number for clang"
+  echo
+  echo "For example: "
+  echo "    toss4_nvcc_clang.sh 12.9.1 90 19.1.3-magic"
+  exit
+fi
+
+COMP_NVCC_VER=$1
+COMP_ARCH=$2
+COMP_CLANG_VER=$3
+shift 3
+
+BUILD_SUFFIX=lc_toss4-nvcc${COMP_NVCC_VER}-${COMP_ARCH}-clang${COMP_CLANG_VER}
+
+echo
+echo "Creating build directory build_${BUILD_SUFFIX} and generating configuration in it"
+echo "Configuration extra arguments:"
+echo "   $@"
+echo
+
+rm -rf build_${BUILD_SUFFIX} >/dev/null
+mkdir build_${BUILD_SUFFIX} && cd build_${BUILD_SUFFIX}
+
+module load cmake/3.23.1
+
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_CXX_COMPILER=/usr/tce/packages/clang/clang-${COMP_CLANG_VER}/bin/clang++ \
+  -DBLT_CXX_STD=c++17 \
+  -C ../host-configs/lc-builds/toss4/nvcc_clang_X.cmake \
+  -DENABLE_CLANGFORMAT=Off \
+  -DCLANGFORMAT_EXECUTABLE=/opt/rocm-5.2.3/llvm/bin/clang-format \
+  -DENABLE_OPENMP=On \
+  -DENABLE_CUDA=On \
+  -DRAJA_ENABLE_NV_TOOLS_EXT=OFF \
+  -DCUDA_TOOLKIT_ROOT_DIR=/usr/tce/packages/cuda/cuda-${COMP_NVCC_VER} \
+  -DCMAKE_CUDA_COMPILER=/usr/tce/packages/cuda/cuda-${COMP_NVCC_VER}/bin/nvcc \
+  -DCMAKE_CUDA_ARCHITECTURES=${COMP_ARCH} \
+  -DENABLE_BENCHMARKS=On \
+  -DCMAKE_INSTALL_PREFIX=../install_${BUILD_SUFFIX} \
+  "$@" \
+  ..
+
+#  -DRAJA_ENABLE_NV_TOOLS_EXT=ON \
+#  -DNVTOOLSEXT_DIR=/usr/tce/packages/cuda/cuda-12.9.1/targets/x86_64-linux/include/nvtx3 \
```

Comment:
@MrBurmark Are these still required? I had an earlier version of this (before our latest RAJA release) where these flags were causing some trouble.

Reply:
Not sure, but why not set ``DRAJA_ENABLE_NV_TOOLS_EXT=ON `` ?